### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: .typos.toml
Comment (attila-lin) [inline .typos.toml:1]: would you mind remove this file

Comment (jayvdb) [reply .typos.toml:1]: This is the config for https://crates.io/crates/typos-cli

I can add CI for it if you like, or happy to remove if you are not interested in this

Comment (jayvdb) [reply .typos.toml:1]: @attila-lin , could you confirm you really dont want this tool to work in this repo ?

Comment (jayvdb) [reply .typos.toml:1]: ok, typos-cli config removed.

**Pull request merged**